README.md

```diff
@@ -1,27 +1,116 @@
 
-This project is a starting point for a Flutter application.
+![Pub DEV](https://img.shields.io/badge/PubDev-0.0.0-blue)
 
-A few resources to get you started if this is your first Flutter project:
+# Recco Flutter Plugin
 
-- [Lab: Write your first Flutter app](https://docs.flutter.dev/get-started/codelab)
-- [Cookbook: Useful Flutter samples](https://docs.flutter.dev/cookbook)
+This repository holds the codebase for the Recco Flutter plugin. Instructions on how to add and use this plugin can be found in the [Usage](#usage) section.
 
-For help getting started with Flutter development, view the
-[online documentation](https://docs.flutter.dev/), which offers tutorials,
-samples, guidance on mobile development, and a full API reference.
+You are more than welcome to install the [APK](#demo) to start playing around with the product, or clone the repository and compile it on your own if you feel like it. 
 
-## Communication between platforms
+However, please be aware that in order to actually integrate it into your own application, you will first need to contact our sales team to initiate the license acquisition process. For that purpose, please send an email to `recco@significo.com`.
 
-This app is using [Pigeon](https://pub.dev/packages/pigeon) lib to make easy the communication between different platforms.
+# Overview
 
-This works as follow:
-- Edit `pigeons/recco_api.dart` file to create the communication API layer between flutter and other native platforms.
-- Run  `./generate_pigeon.sh` to generate the API for the different platform languages. You can edit this file to set options for the generated code.
-- The new generated code will be saved in the output folders specified in this file, in the platform folders (e.g: `android`, `ios`,...).
-- Connect the concrete implementation of the generated API in your native code using the `.setUp()` method (see `MainActivity` as reference).
-- Call the api methods from flutter code to communicate with the native platforms (see `lib/main.dart` as reference, this is the entry point of the app).
+Personalized, and user-friendly approach to health and wellness recommendations based on users' unique interests, motivations, and health behaviors. Whether they are looking to improve their fitness, manage their stress, or simply lead a more fulfilling life. __`Recco` is designed to help them reach their full potential.__
+
+Learn more about Recco SDK:
+
+- [Recco SDK Android][Recco SDK Android]
+- [Recco SDK iOS][Recco SDK iOS]
+
+## Usage
+
+Add this to pubspec.yaml
+
+```yml
+dependencies:
+  recco: 0.0.0
+```
+
+## Initialize Recco in Android
+
+Create an Application.kt class in your Android project, reference it in your Application Manifest.xml file and add the following code:
+
+```kotlin
+// Follow demo example to add Hilt Dependency.
+
+@HiltAndroidApp
+class ShowcaseApp : FlutterApplication() {
+    override fun onCreate() {
+        super.onCreate()
+
+        ReccoApiUI.init(
```

Comment:
can't this be handled as part of `ReccoPlugin`? that way we would expose a unified API for both android and iOS, right?